libkbfs/kbfs_ops.go

```diff
@@ -320,7 +320,11 @@ func (fs *KBFSOpsStandard) getMDByHandle(ctx context.Context,
 		rmd = fbo.getHead(lState)
 	}
 	if rmd != (ImmutableRootMetadata{}) {
-		return rmd, nil
+		if !fbo.identifyDone && getExtendedIdentify(ctx).behavior.AlwaysRunIdentify() {
```

Comment:
Thanks! Changes made, let me know if that is what you had in mind. @strib, for the re-vendor, I had to do it because I added a new "identify mode" type to the `keybase1` protocol. The desktop team wanted their own flag separate from the CLI one we added in there (although they act the same).

 Let me know if you really don't want to do that vendor, and I can back that change out.